luagraphqlparser/lib.c

```diff
@@ -668,7 +668,7 @@ end_visit_fragment_definition(const struct GraphQLAstFragmentDefinition *def, vo
 
 		lua_pushliteral(L, "directives");
 		lua_insert(L, -2);
-		lua_settable(L, -3);
+		lua_settable(L, -4);
```

Comment:
It's a bit inconsistent with other similar code fragments.

I guess we should add an offset for namedType as it was done here:
https://github.com/tarantool/luagraphqlparser/blob/f6ecd5376ec81d8f7a175f53415db4fa0fbf8a0e/luagraphqlparser/lib.c#L743-L748

and use it here:
https://github.com/tarantool/luagraphqlparser/blob/f6ecd5376ec81d8f7a175f53415db4fa0fbf8a0e/luagraphqlparser/lib.c#L767

Reply:
But this value is always the same. And it is the same as in the code just above for filling the 'selectionSet' field.